google/cloud/storage/client.py

```diff
@@ -52,7 +52,6 @@
 from google.cloud.storage.acl import DefaultObjectACL
 from google.cloud.storage.constants import _DEFAULT_TIMEOUT
 from google.cloud.storage.retry import DEFAULT_RETRY
-from google.cloud.storage.retry import DEFAULT_RETRY_IF_METAGENERATION_SPECIFIED
```

Comment:
If `DEFAULT_RETRY_IF_METAGENERATION_SPECIFIED` is only supposed to be used for "write" operations, then we need to update other places, too:  `Bucket.get_blob`, `Bucket.exists`, `Blob.exists` should all use `DEFAULT_RETRY`.